packages/fluentui/react-northstar/src/themes/teams/components/Card/cardStyles.ts

```diff
@@ -16,7 +25,10 @@ const cardStyles: ComponentSlotStylesPrepared<CardStylesProps, CardVariables> =
 
       // TODO: update with latest design spec
       width: pxToRem(300),
-      border: '1px solid black',
+      borderWidth,
```

Comment:
```suggestion
      borderWidth: v.borderWidth,
```

Let's avoid usage of `siteVariables` in styles